parking_permits/models/vehicle.py

```diff
@@ -151,6 +151,10 @@ class Meta:
         verbose_name = _("Vehicle")
         verbose_name_plural = _("Vehicles")
 
+    def save(self, *args, **kwargs):
+        self._is_low_emission = self.is_low_emission
```

Comment:
This will not work if the low emission criteria is changed.

Reply:
We discussed this point in the call and decided that for now, updating the `_is_low_emission` field every time a new order is created should be enough (since this field is currently used only for orders). Resolved in [505ee39](https://github.com/City-of-Helsinki/parking-permits/pull/260/commits/505ee399a8dcacd8a1129d4a66e65bd63b8ef952)

Should be revisited somewhere in the future; this isn't exactly the best solution for this problem.